screens/Home.js

```diff
@@ -67,26 +67,24 @@ const Home = () => {
       }
       
       fetchuserData()
-    }, [])
+    }, [user])
   
 
   const dt =
-    garage.length !== 0 ? new Date(garage[3].nextServiceDate) : new Date();
+    garage.length !== 0 ? new Date(garage[0].nextServiceDate) : new Date();
   const lstDt =
-    garage.length !== 0 ? new Date(garage[3].lastServiceDate) : new Date();
+    garage.length !== 0 ? new Date(garage[0].lastServiceDate) : new Date();
```

Comment:
Changing this to 0 causes an error invalid time, use garage[3] for now till we dynamically render information based on selected car

Reply:
Whenever cars are added to garage, number needs to be updated to ensure clean data

Reply:
can you add selected car to redux, then pull the selected car and store in state e.g. [selectedCar, setSelectedCar] then if there is selectedCar get the nextServiceDate and lastServiceDate

Reply:
the reason for that error is because probably the first car you added to your database has no last service date and as such, it'll be undefined. I did zero because after a car is added the user will now have at least 1 car which is like garage[0]. but I'll try to make it dynamic with redux

Reply:
ok. understood. You can create a new branch for that